Lib/asyncio/events.py

```diff
@@ -672,6 +672,26 @@ def get_event_loop(self):
 
         Returns an instance of EventLoop or raises an exception.
         """
+        if (self._local._loop is None and
+                not self._local._set_called and
+                threading.current_thread() is threading.main_thread()):
+            stacklevel = 2
+            try:
+                f = sys._getframe(1)
+            except AttributeError:
+                pass
+            else:
+                while f:
```

Comment:
Can you add a comment explaining why this loop is needed? (IIUC it's so the warning is attached to a line outside asyncio itself?)

Reply:
dONE.